pynumdiff/tests/test_utils.py

```diff
@@ -110,5 +117,13 @@ def test_simulations(request):
             ax.set_title(title, fontsize=18)
             if i == 5: ax.legend(loc='lower right', fontsize=12)
 
-# def test_evaluate():
-#     return
+
+def test_robust_rme():
+    """Ensure the robust error metric is the same as RMSE for big M, and that it does
+    better in the presence of outliers"""
+    u = np.sin(np.arange(100)*0.1)
+    v = u + np.random.randn(100)
+    assert np.allclose(evaluate.rmse(u, v), evaluate.robust_rme(u, v, M=6))
```

Comment:
Finally added an evaluation code test. It's mildly tricky to see how the `robust_rme` can equal `rmse`, but with big enough `M`, they now produce the same answer. And in the presence of outliers, the robust version is thrown off far less.